components/support/nimbus-fml/src/intermediate_representation.rs

```diff
@@ -439,13 +467,32 @@ impl FeatureManifest {
         self.feature_defs.iter()
     }
 
+    pub(crate) fn iter_imported_files(&self) -> Vec<ImportedClass> {
```

Comment:
a pass by thought about the return types, a "good first bug" could be to turn the `Vec`s into iterators in any `iter_*` functions 🤔 (I know this isn't the only one, so not requesting changes on it here)